pkg/cmd/pulumi/new.go

```diff
@@ -572,6 +576,49 @@ func errorIfNotEmptyDirectory(path string) error {
 
 func validateProjectName(ctx context.Context, b backend.Backend,
 	orgName string, projectName string, generateOnly bool, opts display.Options,
+) error {
+	handleExistingProjectName := func(projectName string) error {
+		surveycore.DisableColor = true
+		surveyIcons := survey.WithIcons(func(icons *survey.IconSet) {
```

Comment:
Feels like we've probably got a few places making up an 'edit'/'continue' survey loop now? Worth finding them all and making a helper in cmdutils?